src/ResourceManager/DataFactories/Commands.DataFactories/Commands.DataFactories.Netcore.csproj

```diff
@@ -29,7 +30,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.DataFactories" Version="4.13.3" />
-    <PackageReference Include="Microsoft.DataTransfer.Gateway.Encryption" Version="3.4.6604.1" />
+    <PackageReference Include="Microsoft.DataTransfer.Gateway.Encryption" Version="3.11.6886.9" />
+    <PackageReference Include="WindowsAzure.Storage" Version="5.0.2" />
```

Comment:
You cannot add a reference to `WindowsAzure.Storage`. We have a shared version as part of our `.targets` file. All modules that use this must use that specific version.

Reply:
@zhangyd2015 Ah, you're right. However, the version used by all modules is `9.3.0`. So, you have to use that version. We're in the process of going to a common library for modules that rely on WindowsAzure.Storage. That issue is: https://github.com/Azure/azure-powershell/issues/7100

Reply:
Updated to `9.3.0`